package/Dockerfile

```diff
@@ -38,10 +39,6 @@ RUN BUILD_DEPS="make ruby-dev build-essential autoconf automake libtool libsnapp
                      libjemalloc1 \
                      ruby \
     && echo 'gem: --no-document' >> /etc/gemrc \
-    && gem install --file Gemfile \
-    && apt-get purge -y --auto-remove \
-                     -o APT::AutoRemove::RecommendsImportant=false \
-                     $BUILD_DEPS \
```

Comment:
I think, this is also must be preserved to make image smaller.

Reply:
Hi, we need it, will update pr and remove WIP after fully test